From 64aca96d9b4a771b63dd536007cddf2413c4d033 Mon Sep 17 00:00:00 2001
From: ishaqsha <shaikhishaque386@gmail.com>
Date: Wed, 7 Mar 2018 16:31:09 +0530
Subject: [PATCH] skjdksadaksjd

---
 1.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/1.c b/1.c
index 01fc60a..605c931 100644
--- a/1.c
+++ b/1.c
@@ -2,3 +2,4 @@ i am added a line
 ************
 saskasakakjdskaj
 haikh ihaque
+dayanad ishaq shaikh patil
-- 
2.7.4

